feat: learn English block 3.1 (#52548)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b11f87368cf463ea4fe46.md

---
id: 657b11f87368cf463ea4fe46
title: Task 8
challengeType: 19
dashedName: task-8
audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
---

<!--
AUDIO REFERENCE:
Anna: That's excellent. Were there any specific projects you'd like to mention?
-->

# --description--

> Questions in the past tense often start with the auxiliary verb `were` or `was`, followed by the subject and then the main verb. This structure helps in asking about past events or experiences. For example, `Were you at the meeting yesterday?` uses this question form. It's important to recognize and understand this form to follow and participate in conversations about past events.

Reviewer: ```suggestion
Questions in the past tense with the verb `to be` are very similar to how you do it in the present tense. The only difference is that you need to remember to conjugate the verb to `were` or `was`, followed by the subject and then the main verb. For example:

 `Were you at the meeting yesterday?` Notice how `were` comes before `you` to for a question. 
 `What was my assignment?` For a specific question, notice how `was` comes after `what` and before `my`. 
```

# --question--

## --text--

How does Anna ask about specific projects in the past tense?

## --answers--

> `Where there any specific projects`

Author: ```suggestion
`Were there any specific projects you'd like to mention?`
```

### --feedback--

`Where` is used for locations. The correct form for asking about past events is `Were`.

---

> `Were there any specific projects`

Author: ```suggestion
`Were there any specific projects you'd like to mention?`
```

---

> `Was there any specific projects`

Author: ```suggestion
`Was there any specific projects you'd like to mention?`
```

### --feedback--

`Was` is used with singular nouns. Since `projects` is plural, `Were` is the correct choice.

---

> `Are there any specific projects`

Author: ```suggestion
`Are there any specific projects you'd like to mention?`
```
